torch/_inductor/codecache.py

```diff
@@ -1547,7 +1546,9 @@ def compile(
         use_absolute_path = False
         if config.is_fbcode():
             ld_command = build_paths.ld()
-            if not cuda and graph.aot_mode:  # Meta internal AOTInductor CPU
+            if (
+                not (device_type == "cuda") and graph.aot_mode
```

Comment:
Based on the comment here, it seems more appropriate to test `device_type == "cpu"` here

Reply:
Sure, thanks.